pokemon_search/src/App.js

```diff
@@ -35,7 +29,13 @@ export default function App() {
       .finally(() => {
         setLoading(false);
       });
-  }, []);
+  }, [offset]);
+
+  const fetchAll = async (urls) => {
+    const res = await Promise.all(urls.map((u) => fetch(u)));
+    const jsons = await Promise.all(res.map((r) => r.json()));
+    setData(jsons);
```

Comment:
We should use try/catch blocks to handle errors when dealing with async/await.

The following are equivalent:

```
SomePromise.then(/* handle response */).catch(/* handle errors */)
```
And

```
try {
  const res = await SomePromise();
  /* handle response */
} catch {
  /* handle errors */
}
```